src/BinDeps.jl

```diff
@@ -182,7 +193,7 @@ module BinDeps
         description::String
         step::SynchronousStepCollection
         Choice(name,description,step) = (s=SynchronousStepCollection();lower(step,s);new(name,description,s))
-    end 
```

Comment:
try this github trick: https://github.com/JuliaLang/BinDeps.jl/pull/136/files?w=1